src/pobox.erl

```diff
@@ -25,6 +25,33 @@
               data = undefined :: undefined | queue() | list()}).
 -endif.
 
+-define(
```

Comment:
Guards where inconsistent and too complex to get right with so many start_link calls so I moved it to a macro. I think it solves the problem.